script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -1155,7 +1155,7 @@ To be honest, I don't feel comfortable talking about the Tohno family.
 [sha:9721676bf7194cebfa87cf8b9a85550c87ddb31a]{
 -- Page 151, Offset 848.
 -- 「違いますよ先輩。遠野は長い休みになると家に居づらいってんで逃げてきただけです。このヤロウ、預けられたコトで有間の人たちに遠慮してんの」
-"That's not it, Senpai. He ran away just because he didn't wanna stay home during summer break. This bastard feels reserved toward the Arimas 'cause he's been left in their care."
+"That's not it, Senpai. He ran away just because he didn't wanna stay home during summer break. This bastard feels a bit guilty about the Arimas being entrusted with him."
```

Comment:
Last sentence feels a bit off to me, maybe just the verb choice (entrusting feels more valued)
> feels a bit guilty about how he was foisted on the Arimas
> feels a bit guilty about imposing on the Arimas

Reply:
"feels a bit guilty about imposing on the Arimas" works. 
Veto first suggestion

Reply:
I think I kept it as 'entrusted' because it gets referenced later down.

I'll see if I can change both.